fairlearn/reductions/moments/conditional_opportunity.py

```diff
@@ -48,6 +48,15 @@ def signed_weights(self, lambda_vec):
         )
         return signed_weights
 
+    def extract_A(self, X, aux_data):
```

Comment:
This obviously needs a lot of work to cope with the different input types we support. For now, it's a sketch